src/oc/web/expo.cljs

```diff
@@ -0,0 +1,64 @@
+(ns oc.web.expo
+  "TODO: Write a couple notes here briefly explaining the bridge."
+  (:require [oc.web.actions.user :as user-actions]
+            [oc.web.utils.user :as user-utils]))
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Native/web bridge primitives
+
+(defn- bridge-call!
+  "Raises an event on the native side of the bridge with name `op` and accompanying `data`.
+  Supported ops are implemented in the open-company-mobile repository."
+  [op data]
+  (let [event (clj->js {:op op :data data})
+        json-event (js/JSON.stringify event)]
+    (js/window.ReactNativeWebView.postMessage json-event)))
```

Comment:
Seems a good place to use a multimethod and specify each action with only a keyword. Like we do in `oc.web.dispatcher/actions` WYT?

Reply:
yeah never mind, i was suggesting the use of multimethods for `bridge-request-push-notification-permission!` and other calls that will be added later since all of them will be using this `bridge-call!`.
Maybe it's not a really good case for multimethods though, never mind for now.